src/systems.rs

```diff
@@ -79,19 +79,17 @@ pub struct CollisionSystem;
 impl<'a> System<'a> for CollisionSystem {
     type SystemData = (
         Entities<'a>,
-        ReadStorage<'a, Projectile>,
         ReadStorage<'a, Transform>,
         ReadStorage<'a, Faction>,
         ReadStorage<'a, Collider>,
-        ReadStorage<'a, Base>,
         Write<'a, Vec<CollisionEvent>>,
     );
 
     fn run(&mut self, data: Self::SystemData) {
-        let (ents, projectiles, transforms, factions, colliders, bases, mut collision_events) = data;
+        let (ents, transforms, factions, colliders, mut collision_events) = data;
 
         // Collision of Projectiles to Enemies
-        for (_projectile, ent, transform, faction, collider) in (&projectiles, &ents, &transforms, &factions, &colliders).join() {
+        for (ent, transform, faction, collider) in (&ents, &transforms, &factions, &colliders).join() {
```

Comment:
Can you add a note somewhere here (or maybe further down) about how each collision will generate two events? One in each direction.